index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+
+    	<meta charset="utf-8">
+    	<meta name="description" content="Ice Times">
+    	<meta name="keywords" content="Hockey, Ice Hockey, Pick-Up, Stick 'n' Shoot">
+
+		<title>Ice Times</title>
+
+		<link rel="icon" type="image/ico" href="images/hockey.png"/>
+
+		<!-- external CSS link -->
+		<link href='http://fonts.googleapis.com/css?family=Economica:400,700,400italic,700italic' rel='stylesheet' type='text/css'>
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+
+    <header>
+      <div class="container">
+
+        <div class="navigation">
+          <ul>
+            <li><a href="grid.html">Rinks</a></li>
+            <li><a href="map.html">Map</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact Us</a></li>
+          </ul>
+        </div>
+
+      </div>
+    </header>
+
+    <div class="main-search">
+      <div class="container">
+
+        <div class="titleTxt">
+          Welcome to Ice Times
+        </div>
+
+        <div class="search">
+          <!-- <h1> Search by rink </h1> -->
+          <input type="titleTxt" id="rinkSearch"  placeholder="Type rink name here"></input>
+          <div class="button" id="rinkSubmit">
```

Comment:
If this button is behaving as a link (taking you somewhere else), semantically it's probably better served as an `<a>` tag.